dotCMS/src/main/java/com/liferay/portal/language/LanguageUtil.java

```diff
@@ -188,6 +188,21 @@ public static Locale getDefaultCompanyLocale(){
         return value;
     }
 
+	/**
+	 * Pretty much the same of get(PageContent, String) but applies if the key exists a sanitizer to avoid escaping issues on the Javascript for the value.
+	 * @param pageContext {@link PageContext}
+	 * @param key {@link String}
+	 * @return String
+	 * @throws LanguageException
+	 */
+    public static String getSanitizeForJS (final PageContext pageContext, final String key)
+			throws LanguageException {
+
+    	final String messageUnsanitize  = get(pageContext,key);
+
+    	return null != messageUnsanitize? messageUnsanitize.replaceAll("'", "\\\\'") : messageUnsanitize;
```

Comment:
Would make sense to use something like **StringEscapeUtils.escapeJavaScript** instead?